### PR TITLE
Add openstack ussuri into support list

### PR DESCRIPTION
**Pull request by jichenjc**
**What this PR does / why we need it**:

Add openstack ussuri into support list (tested)

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

### CODE REVIEW

**Comment by hidekazuna** [comment]
@jichenjc Did you test both HA and non-HA?

**Comment by jichenjc** [comment]
ah... good point, I think you are talking about LB case, correct? 
I tested non-LB case today
I can test LB case or have you tested that?

**Comment by hidekazuna** [comment]
@jichenjc 

> ah... good point, I think you are talking about LB case, correct?

Yes, I am.

> I tested non-LB case today
> I can test LB case or have you tested that?

Unfortunately I only have stable Stein environment and do not have new machine.

**Comment by jichenjc** [comment]
> @jichenjc
> 
> > ah... good point, I think you are talking about LB case, correct?
> 
> Yes, I am.
> 
> > I tested non-LB case today
> > I can test LB case or have you tested that?
> 
> Unfortunately I only have stable Stein environment and do not have new machine.

ok, I have to create LB env (I only have non-LB env now)

**Comment by hidekazuna** [comment]
@jichenjc Did you test lb version?

**Comment by jichenjc** [comment]
@hidekazuna not yet, run out of my time ,if you can help to verify , then it will be helpful
or I will do it in a few days

**Comment by hidekazuna** [comment]
> @hidekazuna not yet, run out of my time ,if you can help to verify , then it will be helpful
> or I will do it in a few days

@jichenjc It's OK, I can wait.

**Comment by jichenjc** [comment]
recheck

**Comment by jichenjc** [comment]
@hidekazuna we by default use U release, so I think we can merge this?

**Comment by jichenjc** [comment]
recheck

**Comment by hidekazuna** [comment]
> @hidekazuna we by default use U release in CI, so I think we can merge this?

@jichenjc If you tested HA version, we can merge this PR.

**Comment by sbueringer** [comment]
@jichenjc any news? Is this PR still active?

**Comment by jichenjc** [comment]
yes, but I am lack of time recently, if you did please help .. thanks @sbueringer

**Comment by sbueringer** [comment]
@jichenjc Sorry I don't have OpenStack Ussuri available, just wanted to ask.
I'm also not entirely sure what exactly we're achieving with our "supported OpenStack versions list".

Almost none of them are continuously tested by us. So basically it comes down to a handful of versions which we 
really know that work, because we're using them all the time. (I assume the actual list is not really in sync with the documented one)

**Comment by jichenjc** [comment]
@sbueringer I agree all you mentioned here, how about let's just say something like
'should work but not tested ...'

**Comment by sbueringer** [comment]
> @sbueringer I agree all you mentioned here, how about let's just say something like
> 'should work but not tested ...'

sounds good to me

File: README.md [hunk 1]
```diff
@@ -53,11 +53,13 @@ This provider's versions are able to install and manage the following versions o
...
-| OpenStack Provider v1alpha1 (release-0.1 branch)| ✓ | ✓ | ✓ | ✓ | + |
-| OpenStack Provider v1alpha2 (v0.2)              | + | ✓ | + | ✓ | + |
-| OpenStack Provider v1alpha3 (v0.3)              | + | + | + | ✓ | ✓ |
+| | OpenStack Pike | OpenStack Queens | OpenStack Rocky | OpenStack Stein | OpenStack Train | OpenStack Ussuri | OpenStack Victoria |
+|-------------------------------------------------|---|---|---|---|---|---|---|
+| OpenStack Provider v1alpha1 (release-0.1 branch)| ✓ | ✓ | ✓ | ✓ | + | + | + |
+| OpenStack Provider v1alpha2 (v0.2)              | + | ✓ | + | ✓ | + | + | + |
+| OpenStack Provider v1alpha3 (v0.3)              | + | + | + | ✓ | ✓ | ✓ | ✓ |
+
+**NOTE:** : There is no formal CI test for specific OpenStack version, the development team has a CI running on latest OpenStack devstack and that covers most of `✓` at above table.
```
Comment (hidekazuna) [inline README.md:62]: This is not true. Our OpenStack version in CI is still Ussuri.
https://github.com/kubernetes-sigs/cluster-api-provider-openstack/blob/master/.zuul/playbooks/run.yaml#L27

Comment (hidekazuna) [reply README.md:62]: CI tests only non HA version. Some OpenStack versions for HA are tested manually.

Comment (jichenjc) [reply README.md:62]: understand .. thanks for point out
so we need some manual test but actually CI is preferred

let me check how we can do and submit some proposal for review

Comment (hidekazuna) [reply README.md:62]: Ideally using CAPI e2e test framework is the best to test HA. I tried but still [in progress](https://github.com/hidekazuna/cluster-api-provider-openstack/tree/e2e). I confirmed it tests HA.

So in this PR, I propose to remove Victoria, and update NOTE:
The development team has a CI using [non HA template](./templates/cluster-template-without-lb.yaml) running on latest OpenStack devstack and that covers most of `✓` at above table. Some versions are tested against HA.

**Comment by jichenjc** [comment]
recheck

**Comment by hidekazuna** [comment]
@jichenjc We can add Victoria now!!

File: README.md [hunk 2]
```diff
@@ -53,11 +53,13 @@ This provider's versions are able to install and manage the following versions o
...
-| OpenStack Provider v1alpha1 (release-0.1 branch)| ✓ | ✓ | ✓ | ✓ | + |
-| OpenStack Provider v1alpha2 (v0.2)              | + | ✓ | + | ✓ | + |
-| OpenStack Provider v1alpha3 (v0.3)              | + | + | + | ✓ | ✓ |
+| | OpenStack Pike | OpenStack Queens | OpenStack Rocky | OpenStack Stein | OpenStack Train | OpenStack Ussuri | OpenStack Victoria |
+|-------------------------------------------------|---|---|---|---|---|---|---|
+| OpenStack Provider v1alpha1 (release-0.1 branch)| ✓ | ✓ | ✓ | ✓ | + | + | + |
+| OpenStack Provider v1alpha2 (v0.2)              | + | ✓ | + | ✓ | + | + | + |
+| OpenStack Provider v1alpha3 (v0.3)              | + | + | + | ✓ | ✓ | ✓ | ✓ |
+
+**NOTE:** : The development team has a CI using [non HA template](https://github.com/kubernetes-sigs/cluster-api-provider-openstack/pull/templates/cluster-template-without-lb.yaml) running on latest OpenStack devstack and that covers most of ✓ at above table. Some versions are tested against [HA template](https://github.com/kubernetes-sigs/cluster-api-provider-openstack/pull/templates/cluster-template.yaml).
```
Comment (hidekazuna) [inline README.md:62]: It is enough to use internal link.

https://github.com/kubernetes-sigs/cluster-api-provider-openstack/pull/templates/cluster-template-without-lb.yaml
-> ./templates/cluster-template-without-lb.yaml

https://github.com/kubernetes-sigs/cluster-api-provider-openstack/pull/templates/cluster-template.yaml
-> ./templates/cluster-template.yaml

Comment (jichenjc) [reply README.md:62]: ok, will do

**Pull request merged**